src/components/side-navigation/side-navigation-item.component.less

```diff
@@ -0,0 +1,77 @@
+ux-side-navigation-item {
+    position: relative;
+    display: block;
+    font-weight: 400;
+    font-size: 0.9375rem;
+
+    &.no-children {
+        &.active {
+            font-weight: 600;
+        }
+    }
+
+    &.collapse[style] {

```

Comment:
What is this for? I don't think we should ever make a selector dependent on the style attribute.